internal/cli/atlas/events/list.go

```diff
@@ -130,6 +130,22 @@ func (opts *ListOpts) NewProjectListOptions() admin.ListProjectEventsApiParams {
 	return p
 }
 
+func (opts *EventListOpts) ValidateMaxAndMinDates() func() error {
```

Comment:
any reason to not do it directly in the NewOrgListOptions and NewProjectListOptions changing those methods to return the error and checking it instead of assuming it was previously validated?

Reply:
if you feel strongly as a prerequisite can you please add a unit test?

Reply:
nope, no strong opinion here, just seemed logic to do this as pre-run to validate the flag format. I'll update